docs/portals/ckan.md

```diff
@@ -1,5 +1,200 @@
+from frictionless import Package
+
+ckan_control = CkanControl(baseurl='https://dados.gov.br', apikey='YOUR-SECRET-API-KEY')
+package = Package(...) # Create your package
+package.publish(control=ckan_control)
+```
+
+# Reading a Catalog
+
+You can download a list of CKAN datasets using the Catalog.
+
+```python tabs=Python
+
+import frictionless
+from frictionless import portals, Catalog
+    
+ckan_control = portals.CkanControl(baseurl='https://dados.gov.br')
+c = Catalog(control=ckan_control)
+```
+
+This will download all datasets from the instance, limited only by the maximum
+number of datasets returned by the instance CKAN API. If the instance returns
+only 10 datasets as default, you can request more packages passing the
+parameter `num_packages`. In the example above if you want to download 1000
+datasets you can do as:
+
+```python tabs=Python
+
+import frictionless
+from frictionless import portals, Catalog
+    
+ckan_control = portals.CkanControl(baseurl='https://dados.gov.br', num_packages=1000)
+c = Catalog(control=ckan_control)
+```
+
+It's possible that when you are requesting a large number of packages from
+CKAN, that some of them don't have a valid Package descriptor according to the
+specifications. In that case the standard behaviour will be to stop downloading
+a raise an exception. If you want to ignore individual package errors, you can
+pass the parameter `ignore_package_errors=True`:
+
+
+```python tabs=Python
+
+import frictionless
+from frictionless import portals, Catalog
+    
+ckan_control = portals.CkanControl(baseurl='https://dados.gov.br', ignore_package_erros=True, num_packages=1000)
+c = Catalog(control=ckan_control)
+```
+
+And the output of the command above will be the CKAN datasets ids with errors
+and the total number of packages returned by your query to the CKAN instance:
+
+```
+Error in CKAN dataset 8d60eff7-1a46-42ef-be64-e8979117a378: [package-error] The data package has an error: descriptor is not valid (The data package has an error: property "contributors[].email" is not valid "email")
+Error in CKAN dataset 933d7164-8128-4e12-97e6-208bc4935bcb: [package-error] The data package has an error: descriptor is not valid (The data package has an error: property "contributors[].email" is not valid "email")
+Error in CKAN dataset 93114fec-01c2-4ef5-8dfe-67da5027d568: [package-error] The data package has an error: descriptor is not valid (The data package has an error: property "contributors[].email" is not valid "email") (The data package has an error: property "contributors[].email" is not valid "email")
+Total number of packages: 13786
+```
+
+You can see in the example above that 1000 packages were download from a total
+13786 packages. You can download other packages passing an offset as:
+
+```python tabs=Python
+
+import frictionless
+from frictionless import portals, Catalog
+    
+ckan_control = portals.CkanControl(baseurl='https://dados.gov.br', ignore_package_erros=True, results_offset=1000)
```

Comment:
type in param name : ignore_package_erros

Reply:
fixed, thanks